contrib/Zipkin/SpanConverter.php

```diff
@@ -44,10 +44,11 @@ private function sanitiseTagValue($value)
 
     public function convert(Span $span)
     {
+        $spanParent = $span->getParent();
         $row = [
             'id' => $span->getContext()->getSpanId(),
             'traceId' => $span->getContext()->getTraceId(),
-            'parentId' => $span->getParent() ? $span->getParent()->getSpanId() : null,
+            'parentId' => $spanParent ? $spanParent->getSpanId() : null,
```

Comment:
You can fix this with what Psalm calls "memoizing simple method calls" setting, but in general you want this form.

Reply:
Alternatively, making classes immutable, which Psalm also supports.